doc.ddoc

```diff
@@ -83,40 +85,45 @@ $(TRANSLATIONS)
 _=
 
 NAVIGATION=
-$(NAVBLOCK_HEADER $(TOCHEADERL index.html, D Programming Language, D $(LATEST)),
-    $(TOCENTRYTH download.html, Download a D compiler, $(SPANC emph,Download), $(CATEGORY_DOWNLOAD $(SUBNAV_DOWNLOAD)))
```

Comment:
This change seems to have removed the download submenu, where users could click to read more about the compiler flags for their specific OS, from the main menu.  Is this intentional?